
Mon, 18 Jun 2018 12:03:29 GMT

COMMAND LINE:
/usr/bin/graalpython fasta.graal-3.graal 2500000

PROGRAM FAILED 


PROGRAM OUTPUT:

Please note: This Python implementation is in the very early stages, and can run little more than basic benchmarks at this point.
Traceback (most recent call last):
  File "fasta.graal-3.graal", line 138, in <module>
    b'main()'
  File "fasta.graal-3.graal", line 125, in main
    b'nprint = sys.stdout.buffer.write'
AttributeError: 'FileIO' object has no attribute buffer
